Dockerfile

```diff
@@ -2,6 +2,7 @@ FROM debian:buster
 MAINTAINER Camptocamp "info@camptocamp.com"
 
 ENV MAPCACHE_VERSION=branch-1-10 \
+    MAPCACHEPATH=mapcache \
```

Comment:
minor but could the name be changed to `MAPCACHE_PATH`?

Reply:
done, thanks